cedar-policy/CHANGELOG.md

```diff
@@ -3,3 +3,568 @@
 The changelog for all the release branches of `cedar-policy` is maintained on
 the `main` branch. You can view the most up-to-date changelog
 [here](https://github.com/cedar-policy/cedar/blob/main/cedar-policy/CHANGELOG.md).
+
+## [3.2.2] - TBA
+
+### Changed
+
+- `unknown()` is no longer a valid extension function if `partial-eval`
+  is not enabled as a feature.
```

Comment:
I also meant you should delete the rest of the added CHANGELOG text 🙂 It's annoying to keep it in sync with `main`.